great-idea/services.html

```diff
@@ -1,107 +1,153 @@
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis. 
+          </p>
+          <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>UX / UI</h2> 
+       <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin vitae tellus
+            rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur
+            feugiat justo quis aliquet. Curabitur cursus et justo eu consequat.
+            Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend.
+            Mauris euismod facilisis iaculis.
+       </p> 
+       <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>Digital Marketing</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin vitae
+            tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>Web Development</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin vitae
+            tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>iOS Development</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin vitae
+            tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+
+        <article>
+          <h2>Android Development</h2> 
+          <p class= learnText>
+            Nulla suscipit interdum diam at maximus. Proin
+            vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+            consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+            consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.
+          </p> 
+          <button>Learn More</butto>
+        </article>
+      </div>
+<hr>
+     
+        <h2>Some Facts About Our Services</h2> 
```

Comment:
The HTML structure here is a bit off here and I'm going to take a stab and say that's the issue with why the flex box isn't working here.  

For the bottom content, all the text should be on the left and image on the right.  To achieve that you need to wrap all the text in one element and have the image as a second element and apply flex to a parent div wrapping the 2, but it seems that you wrapped only some of the service and then added the image to that part too.  That's why flexing it is causing the image to flex with that part of the service text.  In addition, it doesn't seem like you have a parent div for the service/img to apply the flex to either.